pkg/console/subresource/route/route.go

```diff
@@ -88,6 +101,41 @@ func Validate(route *routev1.Route) (*routev1.Route, bool) {
 	return route, changed
 }
 
+func IsCustomRouteSet(operatorConfig *operatorv1.Console) bool {
```

Comment:
I think this will work:
```golang
func IsCustomRouteSet(operatorConfig *operatorv1.Console) bool {
	return operatorConfig.Spec.Route.Hostname) != 0
}
```

Reply:
definitely 👍